docs/content/manual/harvester-rancher/63-configure-k3s-pool-loadbalancer.md

```diff
@@ -0,0 +1,50 @@
+---
+title: 63-Configure the K3s "Pool" LoadBalancer service
+---
+Prerequisite: 
+Already provision K3s cluster and cloud provider on test plan 
+* 59-Create K3s Kubernetes Cluster 
+* 61-Deploy Harvester cloud provider to k3s Cluster 
+
+#### Create Nginx workload for testing
+1. Create a test-nginx deployment with image nginx:latest.
+  ![image](https://user-images.githubusercontent.com/29251855/158512919-a35a079a-aa75-4ce8-bac6-a79438a2e112.png)  
+1. Add pod label test: test.
+  ![image](https://user-images.githubusercontent.com/29251855/158513017-5afc909a-662a-4f4e-b867-2555241a2cbd.png)
+  ![image](https://user-images.githubusercontent.com/29251855/158513105-09ab472b-7cd4-4352-b4e1-84f673ee7088.png)
+
+#### Create a DHCP LoadBalancer
```

Comment:
This should be removed since this is just the pool load balancer

Reply:
Thanks for the check. Remove the unnecessary line